### ISSUE TITLE
OpenEXR support

### ISSUE THREAD

**Issue by piegamesde** [dragon66/icafe#85]
Is it possible to add image support for the OpenEXR format?

**Comment by dragon66** [comment]
Are you talking about read/write support? Seems like there is little/no metadata information for this format.

**Comment by piegamesde** [comment]
Yes, I'd like to read and write them from Java in some way. What do you mean by "no metadata information"?

**Comment by dragon66** [comment]
I got it.  By "no metadata information", I mean there is no extra information embedded in the image such as parameters related to the camera used to take the picture, or author, copyright etc. for this type of image.

**Comment by piegamesde** [comment]
I haven't really looked deeper into the format, but apparently it supports arbitrary metadata tags and values. But I'm more interested into reading and writing the pixel data than the metadata.
